lib/dotcom_web/components/live_components/trip_planner_form.ex

```diff
@@ -48,35 +47,18 @@ defmodule DotcomWeb.Components.LiveComponents.TripPlannerForm do
     <section class="px-10 py-8 lg:px-20 lg:py-12 mb-4 bg-gray-100">
       <.form
         :let={f}
-        class="md:grid md:grid-cols-2 gap-x-8 gap-y-2"
+        class="flex flex-col md:grid md:grid-cols-[1fr_max-content_1fr] gap-x-8 gap-y-2"
         id={@id}
         for={@form}
         method="get"
         phx-submit="save_form"
         phx-change="handle_change"
         phx-target={@myself}
       >
-        <div :for={field <- [:from, :to]} class="mb-1" id="trip-planner-locations" phx-update="ignore">
-          <.algolia_autocomplete
-            config_type="trip-planner"
-            placeholder="Enter a location"
-            id={"#{@form_name}--#{field}"}
-          >
-            <.inputs_for :let={location_f} field={f[field]} skip_hidden={true}>
-              <input
-                :for={subfield <- @location_keys}
-                type="hidden"
-                class="location-input"
-                id={location_f[subfield].id}
-                value={location_f[subfield].value}
-                name={location_f[subfield].name}
-              />
-            </.inputs_for>
-            <.feedback :for={{msg, _} <- f[field].errors} :if={used_input?(f[field])} kind={:error}>
-              <%= msg %>
-            </.feedback>
-          </.algolia_autocomplete>
-        </div>
+        <.search_box name={"#{@form_name}--from"} field={f[:from]} />
+        <.icon class="fill-brand-primary h-6 w-6 rotate-90 self-end md:rotate-0 md:self-center my-4 md:my-0" name="right-left" />
```

Comment:
In the Figma vertically they're way closer together than your screenshot had, maybe you could reduce the `my-4` to `my-2`?

Reply:
Oooh yeah - I think I spaced it correctly before making the form a flexbox, which then used `gap-y-4`, which then added itself to with `my-4`.

Fix incoming...